cdap-ui/cdap-config.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+const parser = require('./server/config/parser.js');
+let cdapConfig;
+
+async function getCDAPConfig() {
```

Comment:
can this function be memoized? Every resolvers are calling this config, but in actuality, this method only needs to be executed once in the beginning of the server startup. The values can then be shared across all the different places.

Reply:
I thought that the variable was only being assigned once, but now I see that is not the case.

Done.